internal/npm_install/generate_build_file.js

```diff
@@ -989,6 +1020,19 @@ alias(
   actual = "//node_modules/${pkg._dir}:${pkg._name}__typings"
 )
 `;
+
+if (types){
```

Comment:
Instead of looking through the node_modules file system for types why don't we look through the package.json instead.
Looking at `node_modules` you might accidentally pickup a package that wasn't installed by the user.
So maybe something like:
```
const typingPackage = pkgJson.dependencies[`@types/${pkg.name}`]
if(typingPackage) {
  // since the typingPackage exists in the pkgJson then add it as deps since it'll be created as a package at some point
}

```